### ISSUE TITLE
test orm

### ISSUE THREAD

**Issue by francescobianco** [MrFibunacci/Framy#15]
I'm so sorry to disturb you, I would kindly ask you if you could test this library in your project! 
https://github.com/javanile/moldable
With the purpose of knowing my project and understanding if it can be integrated into yours!
To thank you for your availability I am observing your project and giving a star

Install via composer
```
composer require javanile/moldable
```

I ask you if we can help each other

**Comment by MrFibunacci** [comment]
A I see you wrote this to like a bunch of PHP framework developer. So I could do this but first I don't want to and second I could probably crash like everything, so thats another reason I don't wanna try some new experimental fucking orm whatever the shit that is, lib(?), framework(?) I really don't know what it is and I don't care at the moment to implement it somewhere. I don't have need for it.

Not meant to an Insult or anything but I really don't want to.  

tldr: No, goodbye.
